src/Stride.CommunityToolkit/Engine/ScriptComponentExtensions.cs

```diff
@@ -22,9 +23,9 @@ public static float DeltaTime(this ScriptComponent scriptComponent)
     /// </remarks>
     /// <param name="scriptComponent"></param>
     /// <returns></returns>
-    public static CameraComponent GetCamera(this ScriptComponent scriptComponent)
+    public static CameraComponent? GetCamera(this ScriptComponent scriptComponent)
```

Comment:
@Doprez, also I made this returned CameraComponent as nullable as it might return a `null`.

Reply:
Sounds good to me!